main/depend.common

```diff
@@ -1417,10 +1415,6 @@ module_fdda_spnudging.o: \
 	../frame/module_wrf_error.o 
 
 
-module_sf_bep.o: \
-	module_sf_urban.o 
-
-
```

Comment:
can you explain why this one is removed?

Reply:
@dudhia This dependency is declared elsewhere in this file, hence redundant.